gradle/libs.versions.toml

```
@@ -9,11 +9,12 @@ caffeineVersion = "3.1.8"
 guavaVersion = "33.2.1-jre"
 hikariCpVersion = "5.1.0"
 jacksonVersion = "2.17.1"
-javalinVersion = "4.6.8"
+javalinVersion = "6.1.7-SNAPSHOT"
 jcipAnnotationsVersion = "1.0_2"
 # This version of Jetty must be the same major version as used by Javalin, please see above.
 # Once Javalin version is upgraded to the latest, this override may be removed.
-jettyVersion = "9.4.54.v20240208"
+jettyVersion = "11.0.16"
+jettyWebsocketServerVersion = "9.4.51.v20230217"
```

```suggestion
jettyWebsocketClientVersion = "9.4.54.v20240208"
```
https://mvnrepository.com/artifact/org.eclipse.jetty.websocket/websocket-client/9.4.54.v20240208